src/PerformanceTests/Transport.V7.RabbitMQ.V5/RabbitMQProfile.cs

```diff
@@ -10,14 +10,15 @@ class RabbitMQProfile : IProfile, INeedPermutation
 
     public void Configure(EndpointConfiguration endpointConfiguration)
     {
-        var cs = ConfigurationHelper.GetConnectionString("RabbitMQ");
+        var cs = ConfigurationHelper.GetConnectionString("RabbitMQ2");
```

Comment:
I would prefer just having a single connection string, but if we have multiple let us create a better connection string name then nr. 2

Reply:
happy with a better name, but let's do that in a different PR